packages/neon/neon_talk/lib/src/pages/main.dart

```diff
@@ -121,10 +121,20 @@ class _TalkMainPageState extends State<TalkMainPage> {
         await Navigator.of(context).push(
           MaterialPageRoute<void>(
             builder: (context) => NeonProvider(
-              create: (_) => TalkRoomBloc(
-                account: account,
-                room: room,
-              ),
+              create: (_) {
+                final roomBloc = TalkRoomBloc(
+                  account: account,
+                  room: room,
+                );
+
+                roomBloc.room.listen((result) {
+                  if (result.hasSuccessfulData) {
+                    bloc.updateRoom(result.requireData);
+                  }
+                });
```

Comment:
Because I don't want to mix the logic of the blocs. I understand why it would be cleaner to put it into the bloc, but then we'd always have to pass the main bloc to each room bloc which I don't really like.

Reply:
I just fear that we do not cancel the listener.
I can not imagine a situation where the main TalkBloc is disosed before the RoomBloc but wanted to make you aware of this.
I'll let you decide if this is fine

Reply:
Hm yes, I'll change it.

Reply:
Ok it also makes sense from a testing perspective because now I can verify that the updated room will be propagated to the main bloc.